crates/rpc/src/jsonrpc/error.rs

```diff
@@ -42,14 +42,16 @@ impl RpcError {
         }
     }
 
-    pub fn message(&self) -> &str {
+    pub fn message(&self) -> String {
```

Comment:
Could consider `cow`. Sidenote: I did want to make the non-internal errors more descriptive e.g. which method wasn't found? Maybe we add a fuzzy string matcher to provide "did you mean xxx" instead 😁

Reply:
Done.